src/middleware/validation.schema.js

```diff
@@ -36,6 +37,26 @@ const followOrUnfollow = {
   }
 };
 
+const title = Joi.string()
+  .min(3)
+  .trim()
+  .required();
+
+const description = Joi.string()
+  .min(3)
+  .trim()
+  .required();
+
+const content = Joi.string()
```

Comment:
content or body? Pick one and make it uniform

Reply:
I am sorry for this oversight. This should be `body`.